keys-component.html

```diff
@@ -2,21 +2,25 @@
   <br-headline br-title="Keys" br-loading="$ctrl.state.loading">
     <ul class="stackable-menu dropdown-menu">
       <li ng-show="$ctrl.isOwner && !$ctrl.hideGenerate">
-        <a class="stackable-cancel"
-          ng-click="$ctrl.modals.showGenerateKeyPair=true"><i class="fa fa-plus"></i> Generate Key Pair</a>
+        <a class="stackable-cancel" ng-click="$ctrl.modals.showGenerateKeyPair=true">
+          <i class="fa fa-plus"></i> Generate Key Pair
+        </a>
       </li>
       <li ng-show="$ctrl.operations.add">
-        <a class="stackable-cancel"
-          ng-click="$ctrl.modals.showAddKey=true"><i class="fa fa-plus"></i> Add Key</a>
+        <a class="stackable-cancel" ng-click="$ctrl.modals.showAddKey=true">
+          <i class="fa fa-plus"></i> Add Key
+        </a>
       </li>
       <li>
-        <a class="stackable-cancel"
-          ng-click="$ctrl.showRevokedKeys=!$ctrl.showRevokedKeys"><i
-          ng-show="$ctrl.showRevokedKeys" class="fa fa-check-square-o"></i><i ng-hide="$ctrl.showRevokedKeys" class="fa fa-square-o"></i> Show Revoked Keys</a>
+        <a class="stackable-cancel" ng-click="$ctrl.showRevokedKeys=!$ctrl.showRevokedKeys">
+          <i class="fa" ng-class="{'fa-check-square-o': $ctrl.showRevokedKeys, 'fa fa-square-o': !$ctrl.showRevokedKeys}">
```

Comment:
Don't need the extra `fa` in the `ng-class` `'fa fa-square-o'` because it's always there via `class="fa"`.